R/parquet.R

```diff
@@ -14,7 +14,23 @@
 #' @name scan_parquet
 #' @rdname IO_scan_parquet
 #' @examples
-#' # TODO write parquet example
+#' #write example file
+#' my_parquet = tempfile(fileext = ".parquet")
```

Comment:
Isn't it necessary to add processing to delete temporary files?
Also, is an extension necessary?